content/en/docs/collector/getting-started.md

```diff
@@ -72,42 +49,62 @@ features such as automatic upgrade handling, `Service` configuration based on
 the OpenTelemetry configuration, automatic sidecar injection into deployments,
 among others.
 
-### Nomad
+## Nomad
 
 Reference job files to deploy the Collector as an agent, gateway and in the
 full demo can be found at
 [https://github.com/hashicorp/nomad-open-telemetry-getting-started](https://github.com/hashicorp/nomad-open-telemetry-getting-started).
 
-### Linux Packaging
+## Linux Packaging
 
-Every Collector release includes DEB and RPM packaging for Linux amd64/arm64
+Every Collector release includes APK, DEB and RPM packaging for Linux amd64/arm64
 systems. The packaging includes a default configuration that can be found at
 `/etc/otelcol/config.yaml` post-installation.
 
 > Please note that systemd is require for automatic service configuration
 
-To get started on Debian systems run the following replacing `v0.44.0` with the
+### Installation
+
+#### APK
```

Comment:
Only headings at level 2 and 3 appear in the auto-generated TOC. Consider replacing these two headings with one at level 3 like this:

> ### APK installation

Then make the corresponding heading changes below.

Reply:
fixed at 0941baf